src/events/guildScheduledEventCreate.js

```diff
@@ -0,0 +1,30 @@
+const { ChannelType } = require('discord.js');
+const { CHANNEL_NAME } = require('../config');
+const { translateLanguage } = require('../languages/index'); // Import translation function
+
+module.exports = {
+  name: 'guildScheduledEventCreate',
```

Comment:
not sure if this event is in the enum of Discord, check if you can import and use it instead

```
const { Events } = require('discord.js');
```